test/dynamo/test_dynamo_fallback.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch._dynamo as dynamo
+import torch._dynamo.test_case as dt
+
+
+class XlaDynamoFallbackTest(dt.TestCase):
+
+  def test_operator_fallback(self):
+
+    @dynamo.optimize("torchxla_trace_once")
+    def fn_fallback(M, mat1, mat2):
+      A = torch.cummin(M, 1)
+      B = torch.add(A[0], mat1)
+      return torch.add(B, mat2)
+
+    M = torch.randn(5, 10, device=xm.xla_device())
+    mat1 = torch.randn(5, 10, device=xm.xla_device())
+    mat2 = torch.randn(5, 10, device=xm.xla_device())
+
+    res = fn_fallback(M, mat1, mat2)
```

Comment:
I think we should
1. Check that `res` match the eager result.
2. Check the counter (try `met.short_metrics_report()` and you should see the fallback counter), I actually don't know what to expect but if there is a fallback I assume the `ExecuteTime` will be 2 instead of 1. You also need to run the `fn_fallback` once to let it compile and then do a clear counter than rerun to see `ExecuteTime` to be 2 instead of 1.

For more counter related stuff, check https://github.com/pytorch/xla/blob/master/torch_xla/debug/metrics.py

Reply:
Will do.

Reply:
If we don't see the mentioned PyTorch's counter, we can also create our own metric when we collect the fallback nodes -- could be useful for debugging/testing.